### ISSUE TITLE
[discuss] new feature, add custom dimensions when use 'event' method

### ISSUE THREAD

**Issue by chux0519** [react-ga/react-ga#114]
It seems there is no way to custom my own dimensions when use 'event' method.
so I just  use `JSON.stringify`  and set my custom data in `label` option, like:
```
ReactGA.event({
        category: pathname,
        action: 'Fetch Data',
        label: JSON.stringify(payload.query)
      })
```

so what if I want to send my own custom dimensions?
I just added few lines in the 'event' method on my own fork branch [line 282](https://github.com/chux0519/react-ga/blob/feature-dimensions/src/index.js)
```
if (args.dimensions) {
        for (var d in args.dimensions) {
          fieldObject[d] = args.dimensions[d];
        }
      }
```

and now I can use dimensions option, like:
```
let dimensions = {}
      Object.keys(payload.query).sort().forEach((dim, index) => {
        dimensions[`dimension${index + 1}`] = payload.query[dim]
      })
      // console.log(dimensions)
      ReactGA.event({
        category: pathname,
        action: 'Fetch Data',
        label: JSON.stringify(payload.query),
        dimensions:dimensions
      })
```

so what about this? Is this necessary?

**Comment by ivxiv** [comment]
Would be great to have support for custom dimensions and metrics with events - how about a pull request for this? 👍

**Comment by dennisat** [comment]
check this PR https://github.com/react-ga/react-ga/pull/152

**Comment by SimeonC** [comment]
Landed in #152
